### PR TITLE
generate the vhd header by ourself.

### PR DESCRIPTION
**Pull request by andyliuliming**
resolving https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/issues/510

### CODE REVIEW

**Comment by andyliuliming** [comment]
@bingosummer  resolving https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/issues/510

File: src/bosh_azure_cpi/spec/unit/vhd_utils_rspec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```
Comment (bingosummer) [inline src/bosh_azure_cpi/spec/unit/vhd_utils_rspec.rb]: The file name should be `vhd_utils_spec.rb`

File: src/bosh_azure_cpi/lib/cloud/azure/disk/vhd_utils.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Bosh::AzureCloud
+  class VHDUtils
+    # Reference Virtual Hard Disk Image Format Specification
+    # http://download.microsoft.com/download/f/f/e/ffef50a5-07dd-4cf8-aaa3-442c0673a029/Virtual%20Hard%20Disk%20Format%20Spec_10_18_06.doc
+    def self.generate_footer(size_in_bytes)
+      footer = {}
+      cookie = +'conectix'
```
Comment (bingosummer) [inline src/bosh_azure_cpi/lib/cloud/azure/disk/vhd_utils.rb]: What does `+` mean?

**Comment by andyliuliming** [comment]
@bingosummer approve?

**Comment by bingosummer** [comment]
Still reviewing.

**Pull request merged**